app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx

```diff
@@ -130,13 +134,12 @@ const SlashSuggestionItem = (props: Props) => {
                 </View>
                 <View style={style.suggestionContainer}>
                     <Text style={style.suggestionName}>{`${suggestionText}`}</Text>
-                    <Text
-                        ellipsizeMode='tail'
-                        numberOfLines={1}
-                        style={style.suggestionDescription}
-                    >
-                        {description}
-                    </Text>
+                    <Markdown
+                        baseTextStyle={style.suggestionDescription}
+                        textStyles={getMarkdownTextStyles(theme)}
+                        blockStyles={getMarkdownBlockStyles(theme)}
+                        value={description}
```

Comment:
do we want to set any of these as true?

* disableHashtags
* disableAtMentions
* disableChannelLink
* disableAtChannelMentionHighlight
* disableGallery

Reply:
After this conversation [here](https://community.mattermost.com/core/pl/tiapzoetg7b33dx8yjji3xkbdo) we will probably remove the markdown support completely from slash commands.